### PR TITLE
Update code quality of all commons and ui classes/methods

### PR DESCRIPTION
**Pull request by JonathanHoshi**
Added header comments of all public classes and methods of commons and ui folders in both main and test.

### CODE REVIEW

**Review by lowkaiwei98 — CHANGES_REQUESTED** [review]
iinw, you have to start the first sentence with a verb.
image

**Comment by JonathanHoshi** [comment]
> iinw, you have to start the first sentence with a verb. image

Can you give me an example of how to write the header for public test methods?

File: src/test/java/seedu/trackermon/commons/core/ConfigTest.java [hunk 1]
```diff
@@ -9,8 +9,14 @@
 
 import org.junit.jupiter.api.Test;
 
+/**
+ * Contains integration tests for {@code Config}.
+ */
 public class ConfigTest {
 
+    /**
+     * Test method to test the string of a default {@code Config} object.
```
Comment (lowkaiwei98) [inline src/test/java/seedu/trackermon/commons/core/ConfigTest.java:18]: Change this to `Tests the string of a default {@code Config} object.` ?

Comment (lowkaiwei98) [reply src/test/java/seedu/trackermon/commons/core/ConfigTest.java:18]: The comments look good, but just to need to change most of them to start from a noun to a verb.

File: src/main/java/seedu/trackermon/commons/core/JarTools.java [hunk 2]
```diff
@@ -4,6 +4,10 @@
 import java.io.UnsupportedEncodingException;
 import java.net.URL;
 
+/**
+ * A functionality class that interacts different based on the running application
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/commons/core/JarTools.java:8]: Change to `Interacts differently depending if the type of running application is a Jar file or an editor.`?

File: src/main/java/seedu/trackermon/storage/JsonFileManager.java [hunk 3]
```diff
@@ -8,6 +8,9 @@
 
 import javafx.stage.FileChooser;
 
+/**
+ * A class to manage the importing and exporting of Json files.
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/storage/JsonFileManager.java:12]: Change to `Manages the importing and exporting of Json files.`?

File: src/main/java/seedu/trackermon/ui/CommandBuilder.java [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 import javafx.beans.property.SimpleStringProperty;
 
+/**
+ * The UI component that is responsible for creating the command information to be displayed in {@code Tableview}.
```
Comment (lowkaiwei98) [inline src/main/java/seedu/trackermon/ui/CommandBuilder.java:6]: Change to `Creates the command information to be displayed in {@code Tableview}.`?

File: src/main/java/seedu/trackermon/commons/core/JarTools.java (hunk 2 above)
Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/commons/core/JarTools.java:8]: Isn't this a class header, the verb rule only applies to methods header?

File: src/main/java/seedu/trackermon/ui/CommandBuilder.java (hunk 4 above)
Comment (JonathanHoshi) [reply src/main/java/seedu/trackermon/ui/CommandBuilder.java:6]: Isn't this a class header, the verb rule only applies to methods header?

File: src/main/java/seedu/trackermon/commons/core/JarTools.java (hunk 2 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/commons/core/JarTools.java:8]: I'm not 100% sure if it violates the coding standard, but we should still make it consistent with the headers from other classes.

File: src/main/java/seedu/trackermon/storage/JsonFileManager.java (hunk 3 above)
Comment (lowkaiwei98) [reply src/main/java/seedu/trackermon/storage/JsonFileManager.java:12]: Is it a lot of them in nouns? if it's not a lot, i think we should just change

**Review by lowkaiwei98 — APPROVED** [review]
lgtm

**Review by Ardentsoul — APPROVED** [review]
LGTM!

**Pull request merged**